### PR TITLE
More efficient handling of huge input VCFs

### PR DESCRIPTION
**Pull request by sigven**
- Reduce huge variant sets (>500,000) prior to R processing by excluding intergenic/intronic variants

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,14 @@ If you want to interrogate germline variants and their relation to cancer predis
 
 ### News
 
+-   *March 2022*: **1.0.1 release**
+
+    -   Fixed bug for huge input sets that cause JSON output crash
+        - huge input variant sets (WGS) are now reduced prior to reporting with R, i.e. exclusion of intronic and intergenic variants, as well as upstream/downstream gene variants
```
Comment (pdiakumis) [inline README.md:26]:
```suggestion
        - huge input variant sets (WGS) are now reduced prior to reporting with R, i.e. exclusion of intronic and intergenic variants, as well as upstream/downstream gene variants ([#178](https://github.com/sigven/pcgr/issues/178)).
```

File: README.md [hunk 2]
```diff
@@ -20,6 +20,14 @@ If you want to interrogate germline variants and their relation to cancer predis
...
+    -   Fixed bug for cases where mutational signature analysis reports > 18 different aetiologies after fitting
```
Comment (pdiakumis) [inline README.md:27]:
```suggestion
    -   Fixed bug for cases where mutational signature analysis reports > 18 different aetiologies after fitting ([#187](https://github.com/sigven/pcgr/issues/187)).
```

File: README.md (hunk 1 above)
Comment (pdiakumis) [reply README.md:26]: Actually, I'd suggest keeping just a super short summary about changes in the README News, and instead focus on more detail in the CHANGELOG. So perhaps just a 'Fixed #123 and #456'.

**Review by pdiakumis — APPROVED** [review]
Looks good!

File: README.md (hunk 1 above)
Comment (sigven) [reply README.md:26]: Good point, Peter. I'll stick to that habit from now on, but will here accept your current suggestions:-)

**Pull request merged**